src/data/zmod/basic.lean

```diff
@@ -328,6 +328,9 @@ def cast_hom (h : m ∣ n) (R : Type*) [ring R] [char_p R m] : zmod n →+* R :=
 lemma cast_sub (h : m ∣ n) (a b : zmod n) : ((a - b : zmod n) : R) = a - b :=
 (cast_hom h R).map_sub a b
 
+lemma cast_neg (h : m ∣ n) (a : zmod n) : ((-a : zmod n) : R) = -a :=
```

Comment:
Should this (and stuff around it) be [norm-cast]?

Reply:
Hmm. I didn't because of the conditions, but I think these actually are reasonable as conditional norm_cast/simp rules.